packages/donation-form/demo/index.html

```diff
@@ -40,6 +42,39 @@
       opacity: 0.01;
       padding-bottom: 0.6rem;
     }
+
+    donation-form-controller.custom-colors {      
```

Comment:
idk this is showing up on the diff, this is already in main https://github.com/internetarchive/iaux-donation-form/blob/main/packages/donation-form/demo/index.html#L44 🤷